### PR TITLE
Add ocp test job for core_bpf

### PR DESCRIPTION
**Pull request by erthalion**
## Description

Latest versions of Collector contains new collection method core_bpf. Add one new test job to exercise it.
Corresponding OSCI PR: https://github.com/openshift/release/pull/38748

## Checklist
- [ ] Investigated and inspected CI test results
~- [ ] Unit test and regression tests added~
~- [ ] Evaluated and added CHANGELOG entry if required~
~- [ ] Determined and documented upgrade steps~
~- [ ] Documented user facing changes (create PR based on [openshift/openshift-docs](https://github.com/openshift/openshift-docs) and merge into [rhacs-docs](https://github.com/openshift/openshift-docs/tree/rhacs-docs))~

## Testing Performed

New collection method is tested on test pipeline in Collector repo.

### CODE REVIEW

File: COLLECTOR_VERSION [hunk 1]
```diff
@@ -1 +1 @@
-3.14.1
+3.14.x-56-gb9058f4fad
```
Comment (janisz) [inline COLLECTOR_VERSION:1]: There is 3.14.x-77 already on
- https://github.com/stackrox/stackrox/pull/5488

Comment (erthalion) [reply COLLECTOR_VERSION:1]: We would still need to bump it at least once (there are some more fixes are coming to the Falco fork), so if you don't mind I'll merge this one now and take care of the consequent version updates.

**Pull request merged**